.golangci.yml

```diff
@@ -44,6 +44,10 @@ linters:
         - '-QF1008' # disable rule 'Omit embedded fields from selector'
         - '-QF1011' # disable rule 'Omit redundant type from variable declaration'
         - '-ST1016' # disable rule 'Use consistent method receiver names'
+      # https://golangci-lint.run/usage/linters/#staticcheck
+      # https://staticcheck.dev/docs/configuration/options/#initialisms
+      initialisms: ["ACL", "API", "ASCII", "CLI", "CPU", "CSS", "DNS", "EOF", "GUID", "HTML", "HTTP", "HTTPS", "ID", "IP", "JSON", "QPS", "RAM", "RPC", "SLA", "SMTP", "SQL", "SSH", "TCP", "TLS", "TTL", "UDP", "UI", "GID", "UID", "UUID", "URI", "URL", "UTF8", "VM", "XML", "XMPP", "XSRF", "XSS", "SDK", "SIP", "RTP", "AMQP", "DB", "TS"]
+
```

Comment:
question: Opinions here? The above is the latest default list for the `staticcheck` initialisms. I've also added `CLI` and `SDK` to the list.

If we don't feel comfortable maintaining our own custom list, then we can remove this and manually enforce words like `CLI` and `SDK`.

Reply:
👍🏻 I support shortening the list to only the initialisms that we use. Since the list is quite short to begin with, I don't think we're losing very much but customizing it. I also like using the separated lines. It makes adding a new item easier to review!

Thanks for the first pass at the new list! I searched the source code and I think you caught everything. I'll merge your commit in but I may add `UTF8` and `XML`, since I can see us referencing those at some point in the future.

Reply:
Commit e6803e5 adds XML and UTF8 😄